### ISSUE TITLE
problem with go imap compilation

### ISSUE THREAD

**Issue by ghost** [emersion/neutron#48]
Hi, 

firstly, when I try:
$ go get -u github.com/emersion/neutron

I get: 

```
# github.com/emersion/go-imap-idle
../go/src/github.com/emersion/go-imap-idle/client.go:28: cannot use res (type *Response) as type responses.Handler in argument to c.c.Execute:
	*Response does not implement responses.Handler (missing Handle method)
../go/src/github.com/emersion/go-imap-idle/response.go:13: undefined: imap.RespHandler
../go/src/github.com/emersion/go-imap-idle/response.go:27: select case must be receive, send or assign recv
../go/src/github.com/emersion/go-imap-idle/server.go:17: undefined: imap.ContinuationResp
# github.com/emersion/go-imap-quota
../go/src/github.com/emersion/go-imap-quota/client.go:28: invalid operation: c.c.State & imap.AuthenticatedState (mismatched types func() imap.ConnState and int)
../go/src/github.com/emersion/go-imap-quota/client.go:46: invalid operation: c.c.State & imap.AuthenticatedState (mismatched types func() imap.ConnState and int)
../go/src/github.com/emersion/go-imap-quota/client.go:56: cannot use res (type *Response) as type responses.Handler in argument to c.c.Execute:
	*Response does not implement responses.Handler (missing Handle method)
../go/src/github.com/emersion/go-imap-quota/client.go:72: invalid operation: c.c.State & imap.AuthenticatedState (mismatched types func() imap.ConnState and int)
../go/src/github.com/emersion/go-imap-quota/client.go:82: cannot use res (type *Response) as type responses.Handler in argument to c.c.Execute:
	*Response does not implement responses.Handler (missing Handle method)
../go/src/github.com/emersion/go-imap-quota/commands.go:100: undefined: utf7.Encoder in utf7.Encoder.String
../go/src/github.com/emersion/go-imap-quota/commands.go:119: undefined: utf7.Decoder in utf7.Decoder.String
../go/src/github.com/emersion/go-imap-quota/responses.go:78: undefined: imap.RespHandler
../go/src/github.com/emersion/go-imap-quota/responses.go:127: undefined: utf7.Decoder
../go/src/github.com/emersion/go-imap-quota/responses.go:155: undefined: imap.RespHandler
../go/src/github.com/emersion/go-imap-quota/responses.go:127: too many errors
```

My environment:

```
Ubuntu 17.10 64bit
go version go1.8.3 linux/amd64
nodejs --version v6.11.4
npm --version 3.5.2
```

secondly, when I try:
cd $GOPATH/src/github.com/emersion/neutron

I get:
bash: cd: /src/github.com/emersion/neutron: No such file or directory

**Comment by emersion** [comment]
Okay, that's why `go dep` is really needed. Will fix this tonight.

**Comment by emersion** [comment]
Seems more complicated than I thought. Apparently macaron.v1 is broken.

**Comment by ghost** [comment]
If I can help in anything, like test something, I would like to do it.

**Comment by emersion** [comment]
Fixed by #50.
